commit L3.1

renvema opened the pull request:
Have a nice day )

Commit:
commit L3.1
Have a nice day )

Sofasmile commented on src/main/java/database/DBHandler.java:
```diff
@@ -0,0 +1,44 @@
+                    .append(DB_PORT)
+                    .append(MY_DB_TEST)
+                    .append(TIME_ZONE);
+            return DriverManager.getConnection(url.toString(), "root", "13frog");
```
```suggestion
            return DriverManager.getConnection(DB_TYPE + DB_HOST 
                     + DB_PORT + MY_DB_TEST + TIME_ZONE, LOGIN, PASSWORD);
```
and remove StringBuilder

Sofasmile commented on src/main/java/dbService/DBService.java:
```diff
@@ -0,0 +1,118 @@
+
+    public UsersDataSet getUser(long id) throws DBException {
+        try {
+            return (new UsersDAO(connection).get(id));
```
```suggestion
            return new UsersDAO(connection).get(id);
```

Sofasmile commented on src/main/java/dbService/DBService.java:
```diff
@@ -0,0 +1,118 @@
+                connection.rollback();
+            } catch (SQLException ignore) {
+            }
+            throw new DBException(e);
```
better move at block catch

Sofasmile commented on src/main/java/dbService/DBService.java:
```diff
@@ -0,0 +1,118 @@
+
+            StringBuilder url = new StringBuilder();
+
+            url.
```
better create constants for url, login and password

Sofasmile commented on src/main/java/dbService/DBService.java:
```diff
@@ -0,0 +1,118 @@
+                    append("user=tully&").          //login
+                    append("password=tully");       //password
+
+            System.out.println("URL: " + url + "\n");
```
do we need to use System.out.println here?

Sofasmile commented on src/main/java/dbService/DBService.java:
```diff
@@ -0,0 +1,118 @@
+
+    public static Connection getH2Connection() {
+        try {
+            String url = "jdbc:h2:./h2db";
```
better create constants for url, name and password

Sofasmile commented on src/main/java/dbService/dao/UsersDaoImpl.java:
```diff
@@ -0,0 +1,42 @@
+
+public class UsersDaoImpl implements UsersDao {
+    private DBHandler dbHandler;
+    private final String REQUEST_SELECT = "select * from users where login=? and password=?";
```
```suggestion
    private final String REQUEST_SELECT = "SELECT * FROM users WHERE login=? and password=?";
```

Sofasmile commented on src/main/java/dbService/dataSets/UsersDataSet.java:
```diff
@@ -0,0 +1,35 @@
+ */
+@SuppressWarnings("UnusedDeclaration")
+public class UsersDataSet {
+    private long id;
```
better use wrapper

Commit:
commit2 L3.1
-remove StringBuilder;
-change request to Upper case;
-remove folder DataSet, because it was from stepik;

Sofasmile commented on src/main/java/dbService/dao/UsersDaoImpl.java:
```diff
@@ -0,0 +1,42 @@
+public class UsersDaoImpl implements UsersDao {
+    private DBHandler dbHandler;
+    private final String REQUEST_SELECT = "SELECT * FROM users WHERE login=? and password=?";
+    private final String REQUEST_INSERT = "INSERT into users(login, password) values(?, ?)";
```
```suggestion
    private final String REQUEST_INSERT = "INSERT INTO users(login, password) VALUES(?, ?)";
```
